generators/app/index.js

```diff
@@ -13,11 +13,19 @@ module.exports = class extends Generator {
       required: false,
       desc: 'Terraform module project name'
     });
+
+    this.option('docker', {
+      type: String,
+      required: false,
+      default: false,
+      desc: 'Include the Dockerfile'
+    });
   }
 
   initializing() {
     this.props = {
-      name: this.options.name || this.appname
+      name: this.options.name || this.appname,
+      docker: ((this.options.docker.toLowerCase() === 'true') || (this.options.docker.toLowerCase() === 'yes')).toString() 
```

Comment:
Is it possible that `this.options.docker` is `null`?

Reply:
No because it's default value is `False` .